src/cobra/core/formula.py

```diff
@@ -15,22 +14,29 @@
 
 
 class Formula(Object):
-    """Describes a Chemical Formula
+    """Describes a Chemical Formula.
```

Comment:
Should be in imperative tone of voice:
```suggestion
    """Describe a chemical formula.
```